modules/account-lib/src/coin/hbar/transaction.ts

```diff
@@ -85,14 +86,28 @@ export class Transaction extends BaseTransaction {
    *
    * @returns {[string, string]} first element is the recipient, second element is the amount
    */
-  private getTransferData(): [string, string] {
+  private getTransferData(): [string, string, string] {
     let transferData;
-    this._txBody.cryptoTransfer!.transfers!.accountAmounts!.forEach((transfer) => {
-      const amount = Long.fromValue(transfer.amount!);
-      if (amount.isPositive()) {
-        transferData = [stringifyAccountId(transfer.accountID!), amount.toString()];
-      }
-    });
+    const tokenTransfers = this._txBody.cryptoTransfer!.tokenTransfers!;
```

Comment:
i know we had this in place, but let's try to avoid force unwrapping optional properties.

alternative is to explicitly check for existence or provide default values.
e.g.
```
const tokenTransfers = this._txBody.cryptoTransfer?.tokenTransfers || [];
```

Reply:
Yeah they used `!` before, I can change to that.

Reply:
For the amount! and token! on line 94 and 99, is there any way to not using ! since we can't set default value for these